### ISSUE TITLE
How to use aarch64-linux-gnu compiler?

### ISSUE THREAD

**Issue by danhambleton** [floooh/fips#223]
So the linux-make-release config uses the usual cc/c++ compilers (as expected). However, I am trying to link PhysX with a fips configured project (on Linux). PhysX requires the aarch64-linux-gnu compiler (or clang). Is there any way to get fips to play nicely with that?

**Comment by floooh** [comment]
Is this a cross-compiling scenario (since aarch64 is ARM), or is this compiler the "default" compiler on that system? If it's the default compiler it should "just work" as long as the command line options are somewhat compatible with gcc's (clang is close enough).

I only tried switching the compiler recently on Ubuntu with "update-alternatives", which basically switches the cc and c++ links from gcc to clang. In a fips project, cmake correctly detected that the compiler was clang instead of gcc and displayed that in the ```./fips gen``` output. There was a warning about an unknown "-Wno-xxx" compiler option, but otherwise building worked.

So basically, if you can somehow switch cc and c++ to the compiler you need, it should work. Unless it's a cross-compiling scenario, in that case you would need a real cmake toolchain file, similar to this one for Android: https://github.com/floooh/fips/blob/master/cmake-toolchains/android.toolchain.cmake (cross-compiling is trickier, because the entire toolchain must be switched over, not just the compiler executables).

**Comment by danhambleton** [comment]
Hey thanks, that worked great.
